File: Assets/Scripts/Player_Name.js
#pragma strict

var m_camera : Camera;
var player_name : String;

function ChangeName(name : String)
{
	var text_component = transform.GetComponent("TextMesh") as TextMesh;
	text_component.text = name;
}

function Update () {
	if(m_camera != null) {
		transform.LookAt(transform.position + m_camera.transform.rotation * Vector3.forward,
			m_camera.transform.rotation * Vector3.up);
	}
}

File: Assets/Scripts/Player_Name.cs
using UnityEngine;
using System.Collections;

public class Player_Name : MonoBehaviour {
	
	public Camera m_camera;
	public string player_name;
	
	public void ChangeName(string name)
	{
		TextMesh text_component = (TextMesh)transform.GetComponent("TextMesh");
		text_component.text = name;
	}
	
	// Update is called once per frame
	void Update () 
	{
		if(m_camera != null) {
			transform.LookAt(transform.position + m_camera.transform.rotation * Vector3.forward,
				m_camera.transform.rotation * Vector3.up);
		}
	}
}
